test/fuzz/lua/test_engine.lua

```diff
@@ -808,7 +847,12 @@ local function index_opts(space, is_primary)
             break
         end
     end
-
+    if is_multikey then
+        -- Overwrite one of the fields with a multikey one.
+        local mk_field_no = math.random(1, #opts.parts)
+        opts.parts[mk_field_no] =
+            gen_multikey_index_part(space:format(), is_nullable_support)
+    end
```

Comment:
I'd rather add multikey index support to the loop above. Like adding `array` type to `tarantool_indices` and so on. It is more straightforward. Currently it is crutchy. It is up to you though as this is testing code.

This way we can increase options space coverage also - we can test when there are more then one multikey part.

Reply:
Discussed f2f, chose your approach - PTAL.